README.rst

```diff
@@ -24,17 +24,6 @@
 TYPO3 Extension ``extension_builder``
 =====================================
 
-----
```

Comment:
Veto. Please leave this in until somebody is found to take over the main development.